blockchainstar12/solidity-proxy
# Add Interaction Script for NFTProxy Contract

## Description

This PR adds an interaction script for the `NFTProxy` contract. The script allows for minting requests to be sent to the `NFTProxy` contract and logs the transaction details.

## Changes

### Interaction Script (`scripts/interact.js`)

- Connects to the deployed `NFTProxy` contract using ethers.js.
- Sends a minting request to the `NFTProxy` contract with the specified token ID, token URI, and extension.
- Logs the transaction details, including the transaction hash when sent and when mined.

## How to Test

1. **Deploy the `NFTProxy` Contract:**
   Ensure the `NFTProxy` contract is deployed to the network you are interacting with. Update the contract address in the script if necessary.

2. **Run the Interaction Script:**
   ```bash
   npx hardhat run --network localhost scripts/interact.js
